src/sql/migrations.rs

```diff
@@ -445,17 +441,6 @@ CREATE TABLE imap_sync (folder TEXT PRIMARY KEY, uidvalidity INTEGER DEFAULT 0,
                 }
             }
         }
-        if exists_before_update {
-            disable_server_delete = true;
-
-            // Don't disable server delete if it was on by default (Nauta):
-            if let Some(provider) = context.get_configured_provider().await?
```

Comment:
This old migration relying on `get_configured_provider()` is modified. This migration was introduced in 2021 and relies on high-level functions: ebccdbbcb901b8363de6c58dfc638d3c675260de
If user migrates from such old version which is very unlikely, then some setting will not be updated, not a big problem.